packages/autocomplete/package.json

```diff
@@ -22,12 +21,16 @@
     "@babel/core": "^7.3.3",
     "@reach/component-component": "^0.1.1",
     "cat-names": "^2.0.0",
+    "pcln-design-system": "^2.5.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-test-renderer": "^16.8.6",
     "styled-components": "^3.4.10",
     "us": "^2.0.0"
   },
+  "peerDependencies": {
+    "pcln-design-system": ">=2.0.0"
```

Comment:
Can we denote peer dependencies to be `>= 2.5.1`, since there were bugs in 2.4.0 & 2.5.0?

Reply:
My thinking here was that 2.0.4 was the former dependency I had replaced here, so technically any version between 2.0.0 and 2.5.1 should still be usable, aside from 2.4.0 and 2.5.0

Reply:
True also for functional versions >=2.0.0 that are not 2.4.0 & 2.5.0. A discussion also happened to unpublish the broken versions, which would resolve this concern of peer dev.

Therefore LGTM. Thanks.